compose.prod.yml

```diff
@@ -0,0 +1,113 @@
+services:
+  www:
+    # To use a prebuilt image from ghcr.io comment out the "image" and "pull_policy" lines
+    # You can also a tag eg.: "v1.0.0" instead of "latest"
+    #image: "ghcr.io/mbinorg/mbin:latest-caddy"
+    #pull_policy: never
+    ports:
+      - 80:80
```

Comment:
Do not reuse port `80` on the host machine. 99.999% of the time this port is taken already by either Nginx or Apache2. So you should bind this port to another port like `8008` we had before (on purpose). 

_Especially_ on production you will need to add a reverse proxy anyways (eg. for SSL). Exposing port 80 like this will not work together with Nginx or Apache2, while it's actually a requirement.

Reply:
That's highly dependent on the individual. I consider this to be the equivalent of the bare metal. Once the bare metal steps are run through, I assume it's possible to access mbin on http://mydomain.tld. After following the docker prod guide, the user would not (http://mydomain.tld:8008).

Someone using a reverse proxy would probably see the error from docker compose that a port cannot be allocated and modify the port. By setting it to `8008`, you're making the assumption that 0.0001% of admins have a reverse proxy. Since caddy is already the chosen reverse proxy, it has [HTTPS by default](https://caddyserver.com/docs/automatic-https), so theoretically, opening port `443`, should let caddy provision those certs.

In any case, I modified it to `8008`.